test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/xunit.runner.json

```diff
@@ -0,0 +1,4 @@
+﻿{
```

Comment:
Is there a way to reference the file from a single place instead of duplicating it?

https://xunit.net/docs/configuration-files

Reply:
added to our common.props and moved our single file to there. Updated references to point to build/file location